rotkehlchen/api/v1/encoding.py

```diff
@@ -1123,3 +1123,11 @@ class AssetIconsSchema(Schema):
         validate=webargs.validate.OneOf(choices=('thumb', 'small', 'large')),
         missing='thumb',
     )
+
+
+class GraphQuerySchema(Schema):
+    is_graph_query = fields.Boolean(missing=True)
```

Comment:
As discussed yesterday this whole "is_graph_query" thing should be removed from the entire chain of calls. Since all tests pass though we can merge and fix this later.